src/lib/models.ts

```diff
@@ -1,4 +1,6 @@
-import { Manifest } from "ryuu-client/lib/models";
+import { Manifest as RCManifest } from "ryuu-client/lib/models";
+
+export type Manifest = RCManifest;
```

Comment:
Does this intend to allow access to the manifest from apps in the future?

Reply:
It was actually a TS error since Manifest was not getting re-exported from here but referenced in another file. Either way, it should allow to re-use this library to import the interface, yes

Reply:
Oh, and actually, I might also open a patch PR for ryuu-client since the def for Manifest didn't seem accurate from what I saw